armi/reactor/tests/test_components.py

```diff
@@ -810,6 +914,13 @@ def test_dimensionThermallyExpands(self):
 
 
 class TestSquare(TestShapedComponent):
+    """Test square shaped component.
+
+    .. test:: Square shaped component
+        :id: T_ARMI_COMP_SHAPES3
+        :tests: R_ARMI_COMP_SHAPES
+    """
+
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.